packages/block-library/src/navigation/theme.scss

```
@@ -7,5 +7,5 @@
 
 // Overrides generic ".entry-content li" styles on the front end.
 .wp-block-navigation-link.wp-block-navigation-link {
-	margin: 0;
+	margin: 0 8px 0 0;
```

Same as above, it would probably be best to use `em` values.

```suggestion
	margin: 0 0.5em 0 0;
```